closed/OpenJ9.gmk

```diff
@@ -457,45 +484,16 @@ $(J9JCL_SOURCES_DONEFILE) : $(AllJclSource)
 		JAVA_HOME=$(BOOT_JDK) \
 		preprocessor
 	@$(ECHO) Generating J9JCL sources
-	$(BOOT_JDK)/bin/java \
-		-cp "$(call FixPath,$(JPP_JAR))" \
-		-Dfile.encoding=US-ASCII \
-		com.ibm.jpp.commandline.CommandlineBuilder \
-			-verdict \
-			-baseDir "$(call FixPath,$(OPENJ9_TOPDIR))/" \
-			-config SIDECAR18-SE-OPENJ9 \
-			-srcRoot jcl/ \
-			-xml jpp_configuration.xml \
-			-dest "$(call FixPath,$(JPP_DEST))" \
-			-tag:define "$(subst $(SPACE),;,$(sort $(JPP_TAGS)))"
+	$(call RunJPP, SIDECAR18-SE-OPENJ9-CLOSED, $(TOPDIR)/closed/adds/jdk, $(JPP_DEST)) \
```

Comment:
See suggestion for a generic configuration https://github.com/eclipse-openj9/openj9/pull/14982/files#r862138661.

Reply:
Updated according to https://github.com/eclipse-openj9/openj9/pull/14982